vpr/src/place/place.cpp

```diff
@@ -1378,7 +1378,7 @@ static e_move_result try_swap(const t_annealing_state* state,
         //
         //Also find all the pins affected by the swap, and calculates new connection
         //delays and timing costs and store them in proposed_* data structures.
-        int num_nets_affected = find_affected_nets_and_update_costs(
+        find_affected_nets_and_update_costs(
```

Comment:
I think if this function call is written in two lines, it would be more readable:

```
find_affected_nets_and_update_costs(place_algorithm, delay_model, criticalities,
                                    blocks_affected, bb_delta_c, timing_delta_c);
```